lib/ansible_tower_client/collection.rb

```diff
@@ -11,37 +11,47 @@ def all
     end
```

Comment:
collecting or mapping the enumerator will force it and immediately load everything.